modules/tile-converter/src/i3s-converter/helpers/load-3d-tiles.ts

```diff
@@ -65,3 +65,32 @@ export const loadTile3DContent = async (
 
   return tileContent;
 };
+
+/**
+ * Fetch 3DTiles tile content
+ * @param sourceTileset - source tileset that contain the tile
+ * @param sourceTile - source tile JSON metadata
+ * @param globalLoadOptions - tileset-wide load options
+ * @returns ArrayBuffer of loaded content
+ */
+export const fetchTile3DContent = async (
+  sourceTileset: Tiles3DTilesetJSONPostprocessed | null,
+  sourceTile: Tiles3DTileJSONPostprocessed,
+  globalLoadOptions: Tiles3DLoadOptions
+): Promise<ArrayBuffer | null> => {
+  const isTileset = sourceTile.type === 'json';
```

Comment:
Nit: Wouldn't it be nicer to preprocess a bit more so that the information in the postprocessed Tiles3D is more "semantic":
```suggestion
  const isTileset = sourceTile.type === 'tileset';
  // or
  const isTileset = sourceTile.isTileset;
```

Reply:
It is calculated in Tiles3DLoader. As I could see it is the file extension.